test/builtin_dataset_mocks.py

```diff
@@ -217,7 +218,7 @@ def mnist(info, root, config):
     )
 
 
-DATASET_MOCKS.update({name: DatasetMock(name, mnist) for name in ["fashionmnist", "kmnist"]})
+# DATASET_MOCKS.update({name: DatasetMock(name, mnist) for name in ["fashionmnist", "kmnist"]})
```

Comment:
This is the only instance where mocks are registered without using the decorator. Thus, we disable them manually.